Patch by MSP-Greg
test is intermittent on 'busy' Appveyor, increase timeout
diff --git a/bootstraptest/test_io.rb b/bootstraptest/test_io.rb
index 89c00d0b88..b5d617264b 100644
--- a/bootstraptest/test_io.rb
+++ b/bootstraptest/test_io.rb
@@ -30,7 +30,7 @@
   end
 }, '[ruby-dev:32566]'
 
-assert_finish 1, %q{
+assert_finish 2, %q{
   r, w = IO.pipe
   Thread.new {
     w << "ab"
